CONTRIBUTING.rst

```diff
@@ -464,9 +487,97 @@ Step 5: Pass PR Review
 Note that committers will use **Squash and Merge** instead of **Rebase and Merge**
 when merging PRs and your commit will be squashed to single commit.
 
-You need to have review of at least one committer (if you are committer yourself, it has to be
+When the maintainer starts conversations it is expected that you respond to questions, suggestions, doubts,
```

Comment:
```suggestion
When a reviewer starts a conversation it is expected that you respond to questions, suggestions, doubts,
```
I think the check for the necessity of resolved conversations would apply to all conversations irrespective of it being from a maintainer or not, right?

Reply:
Good point